### PR TITLE
Added aarch64 support.

### PR DESCRIPTION
**Pull request by njordan64**
Updated to work with unclutter arguments and enabled shutdown in the xinitrc file.

### CODE REVIEW

File: retroarch-standalone-service/retroarch-standalone.cfg [hunk 1]
```diff
@@ -4,7 +4,7 @@ menu_driver = "xmb"
 menu_show_core_updater = "true"
 assets_directory = "/usr/share/retroarch/assets"
 cache_directory = "~/.cache"
-libretro_directory = "~/.config/retroarch/cores"
+libretro_directory = "/usr/lib/libretro"
```
Comment (julianxhokaxhiu) [inline retroarch-standalone-service/retroarch-standalone.cfg]: Can you please revert this line?

File: retroarch-standalone-service/retroarch-standalone.xinitrc [hunk 2]
```diff
@@ -6,10 +6,18 @@ xset -dpms         # disable DPMS (Energy Star) features.
...
 
 # Clean old debug logs
 rm /home/retroarch/debug.log 2>&1 >&/dev/null
 
 # Autostart Retroarch
-exec /usr/bin/retroarch --appendconfig=/etc/retroarch-standalone.cfg -v 2>&1 >&/home/retroarch/debug.log
+exit_code=1
+while [ "$exit_code" -ne "0" ]
+do
+  /usr/bin/retroarch --appendconfig=/etc/retroarch-standalone.cfg -v --log-file /home/retroarch/debug.log
```
Comment (julianxhokaxhiu) [inline retroarch-standalone-service/retroarch-standalone.xinitrc:18]: There's a report from a user saying that now RetroArch has built-in logging interface. If it hooks up into journald we could remove the redirect to the log-file :) What do you think?

File: retroarch-standalone-service/retroarch-standalone.xinitrc [hunk 3]
```diff
@@ -6,10 +6,18 @@ xset -dpms         # disable DPMS (Energy Star) features.
...
+  exit_code=$?
+done
+
+sudo shutdown -h now
```
Comment (julianxhokaxhiu) [inline retroarch-standalone-service/retroarch-standalone.xinitrc]: Does this work out of the box when we install the package? Or does it require some more work on the user end? Maybe we can add a check if:
1) `sudo` exists in the platform and
2) if the user has sudo rights

If both are true, than a shutdown could be issued.

Thank you in advance :)

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]

File: retroarch-standalone-service/retroarch-standalone.xinitrc (hunk 2 above)
Comment (njordan64) [reply retroarch-standalone-service/retroarch-standalone.xinitrc:18]: I'm not aware of anything builtin to RetroArch for logging to journald. There is a config setting `log_to_file` but I am not having any luck when testing it.

File: retroarch-standalone-service/retroarch-standalone.xinitrc (hunk 3 above)
Comment (njordan64) [reply retroarch-standalone-service/retroarch-standalone.xinitrc]: I'm thinking a better approach here is to try to run another script in the retroarch user's directory if the file is present. Something like
```
if [ -x "/home/retroarch/.xinit_exit" ]
then
  /home/retroarch/.xinit_exit
fi
```

Comment (julianxhokaxhiu) [reply retroarch-standalone-service/retroarch-standalone.xinitrc]: I like the idea, works better and you can customize its behavior easily :)

**Comment by julianxhokaxhiu** [comment]
Thanks for reverting the patch. Can you please squash your commits into one only?

File: retroarch-standalone-service/retroarch-standalone.xinitrc (hunk 2 above)
Comment (julianxhokaxhiu) [reply retroarch-standalone-service/retroarch-standalone.xinitrc:18]: Let's keep it as it is then :) thanks

**Comment by njordan64** [comment]
I've squashed the commits.

**Comment by julianxhokaxhiu** [comment]
Thank you!

**Pull request merged**